CONTRIBUTING.md

```diff
@@ -6,10 +6,11 @@ We encourage you to read this project's CONTRIBUTING policy (you are here), its
 
 ## How to contribute to U.S. Forms System
 
-- Report bugs and request features via [Github Issues](https://github.com/usds/us-forms-system/issues). You can also ask the core team questions and make suggestions by creating an issue.
-- If you prefer email to issues, you can reach us at [forms@lists.usds.gov](mailto://forms@lists.usds.gov) with your questions and comments. We're always open to suggestions.
-- Create a [Github pull request](https://help.github.com/articles/creating-a-pull-request/) with new functionality or fixing a bug.
-- Improve or create documentation for U.S. Forms System.
+- When contributing to this repository, please first discuss the change you wish to make via issue or email with the team mananaging this repository before making a change.
+  - You can report bugs and/or request features via [Github Issues](https://github.com/usds/us-forms-system/issues). You can also ask the team questions and make suggestions by creating an issue. 
+  - If you prefer email to issues, you can reach us at [forms@lists.usds.gov](mailto://forms@lists.usds.gov) with your questions and comments. We're always open to suggestions.
+- Create a [Github pull request](https://github.com/usds/us-forms-system/blob/master/.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md) with new functionality or fixing a bug.
+- Help improve our documentation for U.S. Forms System (for any changes that you make that require updates to the documentation, kindly help us update that accordingly.)
```

Comment:
> Help improve our documentation for U.S. Forms System (for any changes that you make that require updates to the documentation, kindly help us update that accordingly.)

Maybe:

"Help improve our documentation for U.S. Forms System. If you propose changes that require updates to the documentation, kindly help us update that accordingly in your pull request."